src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -7,15 +7,35 @@
 !
 module mpas_init_atm_gwd
 
+   use iso_c_binding, only : c_char, c_int, c_float, c_ptr, c_loc
+
    use mpas_derived_types, only : MPAS_LOG_ERR
    use mpas_framework
    use mpas_timekeeping
    use mpas_log, only : mpas_log_write
+   use mpas_c_interfacing, only : mpas_f_to_c_string
 
    public :: compute_gwd_fields
 
    private
 
+   interface
+      subroutine read_geogrid(fname, rarray, nx, ny, nz, &
+                             & isigned, endian, scalefactor, wordsize, status) bind(C)
```

Comment:
Stylistically, in MPAS we tend to only put the `&` continuation character at the end of the line to be continued. For consistency, it may be nice to remove the beginning `&` character here and anywhere else this occurs (e.g., in the comments at the top of `read_geogrid.c`).